src/view/reminder/commons/reminder-time-options.ts

```diff
@@ -3,79 +3,196 @@
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
-/* eslint-disable @typescript-eslint/explicit-function-return-type */
-/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+
 import { TFunction } from 'i18next';
+import { filter } from 'lodash';
+import moment from 'moment';
+import { useMemo } from 'react';
+import { EventType } from '../../../types/appointment-reminder';
+
+type ReturnType = Array<{
+	id: string;
+	label?: string;
+	click?: () => void;
+	value: number;
+	type?: string;
+}>;
+export const useGetReminderItems = (
+	t: TFunction,
+	setSnooze: (arg1: number, arg2?: boolean) => void,
+	event: EventType
+): ReturnType => {
+	const diff = moment(event?.resource?.alarmData?.[0]?.alarmInstStart).diff(moment(), 'seconds');
```

Comment:
this could be memoized too

Reply:
ya sorry it just skip out of mind.Thanks for pointing it out